### ISSUE TITLE
Add screenshots to readme

### ISSUE THREAD

**Issue by EmmanuelMess** [orangeduck/Corange#49]
It would be nice to show what's possible.

**Comment by orangeduck** [comment]
Good idea - if you want to take some and add them I'm be more than happy to merge it in 👍

**Comment by fedor-elizarov** [comment]
To avoid a large number of pictures, you can make a collage of pictures of different demonstrations (this should be neat and beautiful) in one picture and make a picture as a link to a demo video.
Maybe like this ->  https://i.pinimg.com/originals/73/6a/73/736a7375c4f9245c71a842946a7f230f.jpg
 or this https://i.pinimg.com/564x/be/72/c4/be72c466d5678bf1f734e83b3bae4380.jpg  maybe other best varian

**Everyone who reads this - suggest your variant, Daniel will choose the best image**


P.S. I can’t, my computer mouse is broken, and part of the keyboard is very inconvenient for working with pictures :D
